tests/odov3/odo_test.go

```diff
@@ -61,30 +61,27 @@ func TestOdo(t *testing.T) {
 	// more at: https://onsi.github.io/ginkgo/#dynamically-generating-specs
 	g := NewGomegaWithT(t)
 
-	// make stacks path absolute, it is required because later we are using it from the random directory.
-	stacksDir, err := filepath.Abs(stacksDir)
-	g.Expect(err).To(BeNil())
+	cmd := exec.Command("bash", "../get_changed_stacks.sh")
```

Comment:
Hardcoding this script here will make it impossible to run tests locally with uncommitted changes. 

It might be better to call this script outside of the tests and pass the list of stacks that should be tested as an argument similar to how `stacksDir` is handled.